src/CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ set(mgclient_src_files
         mgsession-encoder.c
         mgtransport.c
         mgvalue.c)
+if(${WIN32})
```

Comment:
Why not just WIN32 instead of ${WIN32}?

Reply:
:+1: That's my bash habit, which doesn't make much sense except consistency. But, I think I'll introduce `MGCLIENT_...` CMake variables because otherwise, I would have to repeat CMake specific logic.